### ISSUE TITLE
New login password is reset if there are any invalid fields in the form.

### ISSUE THREAD

**Issue by sebleedelisle** [ArthurGuy/BBMembershipSystem#167]
Would be nice if the password wasn't reset. It's a bit annoying. Also same with the photo. :)

**Comment by mikestreety** [comment]
I believe this is a security hole if you do send the password back :smile: 

http://ux.stackexchange.com/a/20447

**Comment by sebleedelisle** [comment]
Yeah I was imagining a solution like the one outlined in that answer : 

"If server-side validation fails, store the hashed password in a session variable and set the password field's value to something like ******** (literally, 8 asterisks). When the form is resubmitted with password ********, use the stored password, otherwise use whatever was submitted (in the unlikely event the user decided to change their password before resubmitting)."

**Comment by mikestreety** [comment]
Ah, makes sense! I originally read that completely differently
